solr-ingest/src/test/java/edu/unc/lib/dl/data/ingest/solr/action/UpdateTreeSetActionTest.java

```diff
@@ -177,21 +147,28 @@ public void testNotChildSetRequest() throws Exception {
 
     @Test(expected = IllegalArgumentException.class)
     public void testNoChildrenRequest() throws Exception {
-        when(request.getChildren()).thenReturn(Arrays.asList());
+        request = new ChildSetRequest(null, asList(),
+                IndexingActionType.ADD, USER);
 
         action.performAction(request);
     }
 
-    private ContentContainerObject makeContainerObject() {
+    protected ContentContainerObject makeContainer() {
```

Comment:
why is this protected instead of private?

Reply:
I forgot to use the helper class here, I've updated the test to use it now.